.github/workflows/tests.yml

```diff
@@ -61,3 +61,35 @@ jobs:
           -sdk ${{ matrix.sdk }} \
           -destination ${{ matrix.destination }} \
           test-without-building
+
+  swift-button-functional-test:
+    runs-on: macOS-latest
+    defaults:
+      run:
+        working-directory: Samples/Swift/DaysUntilBirthday
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Create xcconfig file
+      shell: bash
+      env:
+        EMAIL_SECRET : ${{ secrets.EMAIL_SECRET }}
+        PASSWORD_SECRET : ${{ secrets.PASSWORD_SECRET }}
+      run: |
+        echo EMAIL_SECRET = $EMAIL_SECRET$'\n'PASSWORD_SECRET = $PASSWORD_SECRET >> DaysUntilBirthdayUITests\(iOS\)/Credentials.xcconfig
```

Comment:
Did you explore a means of injecting the credentials without writing them to disk, such as using compiler flags?

Reply:
I have tested adding an environment variable to the iOS scheme's Test action. I _think_ we could pass the secrets in there like so (I have gotten this to work locally):

```
xcodebuild \
          -project Samples/Swift/DaysUntilBirthday/DaysUntilBirthday.xcodeproj \
          test-without-building \
          -scheme DaysUntilBirthday\(iOS\) \
          -sdk iphonesimulator \
          -destination 'platform=iOS Simulator,name=iPhone 11' \
          SECRET_EMAIL=$EMAIL_SECRET
```

However, two things stand out to me: 

1. As things stand now, the output for `xcodebuild` print the value of the secrets. Per a local test, it'll look something like this:

```
Build settings from command line:
    FOO_BAR = testValue
    SDKROOT = iphonesimulator15.4
```

I'm guessing that we can have less logged here, but then perhaps that will make debugging more difficult on GitHub.

2. Testing locally will require using `xcodebuild` from the command line, or setting up environment variables with hardcoded values for the iOS scheme's Test action. This will either mean not using Xcode directly for testing locally, or require a bit more setup (IMO).

Reply:
Resolving as I was able to use environment variables passed in through `xcodebuild`.